weapon-config.inc

```diff
@@ -3526,18 +3532,20 @@ public OnPlayerWeaponShot(playerid, weaponid, hittype, hitid, Float:fX, Float:fY
 				new Float:health;
 
 				GetVehicleHealth(hitid, health);
+				if(health >= 249.0) { //vehicles start on fire at 250 or under so theres no need to check once the vehicle is at or below 250
```

Comment:
I orignaly had it so it would stop at 250 and during testing i found out about that. Which is the reason why i set it to 249. Also the reason i do this is so the timer doesn't get ran twice.